app/src/main/java/com/github/se/cyrcle/ui/theme/atoms/Button.kt

```diff
@@ -386,8 +386,7 @@ fun GoogleSignInButton(onClick: () -> Unit) {
       shape = RoundedCornerShape(50),
       border = BorderStroke(1.dp, Color.LightGray),
       modifier =
-          Modifier.padding(start = 16.dp, end = 16.dp)
```

Comment:
Why did you remove this ?

Reply:
Yeah, good question, plus it's used in a lot of places so need to be sure on this

Reply:
That's probably a mistake then, I will put it back